services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/AbstractPushMonitor.java

```diff
@@ -87,6 +94,9 @@ public AbstractPushMonitor(
     this.clusterLockManager = clusterLockManager;
     this.aggregateRealTimeSourceKafkaUrl = aggregateRealTimeSourceKafkaUrl;
     this.activeActiveRealTimeSourceKafkaURLs = activeActiveRealTimeSourceKafkaURLs;
+    this.helixAdminClient = helixAdminClient;
+    this.helixClientThrottler =
+        new EventThrottler(10, 5000, "push_monitor_helix_client_throttler", false, EventThrottler.BLOCK_STRATEGY);
```

Comment:
Why not use the default window size: 30 seconds?

Reply:
Sure.